packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -195,7 +195,17 @@ class _ProductListPageState extends State<ProductListPage> {
                   );
                   if (dismissible) {
                     return Dismissible(
-                      background: Container(color: colorScheme.background),
+                      //Delete Animation Added
+                      direction: DismissDirection.endToStart,
+                      background: Container(
+                        alignment: Alignment.centerRight,
+                        padding: const EdgeInsets.only(right: 30),
```

Comment:
if you switch to a dark theme you will see that the red color container looks bigger than the list tile ,
maybe a margin around the container fix that, you can try the suggested changes

Reply:
I have added margin of 14 to bottom and top.